docs/source/examples/external_sp2.py

```diff
@@ -0,0 +1,19 @@
+"Can be found in gpkit/docs/source/examples/external_sp.py"
+import numpy as np
+from gpkit import Variable, Model
+
+x = Variable("x")
+
+
+def y_ext(self, x0):
```

Comment:
I don't know! It's not anti-pythonic to do this, but is it confusing? I wanted to make sure people weren't just sticking arbitrary constraints into these. :p